src/taxpy/assets/asset.py

```diff
@@ -11,7 +11,17 @@ def __init__(self, name: str, life: int, value: float):
         :param value: The value of the asset ($).
         """
         self.name = name
-        self.LIFE = life
+        # self.LIFE = life
+        self._life = life
```

Comment:
I don't want the variable to be private, I just want to show that it doesn't change.